modules/hub/hub/features/karma/views.py

```diff
@@ -41,6 +42,24 @@ def post(self):
         post: Post = post_id and Post.query.filter_by(id=post_id).first_or_404()
 
         available, reason = is_user_can_rate_post(current_user, post)
+        if "Users" in request.form:
+            users = get_all_users_and_change_rated_post(post)
+            if users:
+                likes = {}
+                dislikes= {}
+                for user in users.keys():
+                    if users[user]>0:
+                        likes[user]=users[user]
+                    else:
+                        dislikes[user]=users[user]
```

Comment:
```suggestion
                for user, change in users:
                    if change>0:
                        likes[user]=change
                    else:
                        dislikes[user]=change
```

А то тут какой-то c-style у тебя)

Reply:
Done